### ISSUE TITLE
Page "http://franom.fraport.de/franom.php" no longer shows planes after update to 1.0.9.2

### ISSUE THREAD

**Issue by xformerfhs** [andryou/scriptsafe#293]
After update to V1.0.9.2 the web site http://franom.fraport.de/franom.php no longer shows planes. The site worked with V1.0.9.1. Allowed sites are "**.google.com", "**.googleapis.com", "**.fraport.de" and "**.fraport.com".

**Comment by andryou** [comment]
Thanks for the report, do you have "Block Browser Plugin Enumeration" under
Fingerprint Protection enabled?

**Comment by xformerfhs** [comment]
Hello, Andrew,

   no, „Block Browser Plugin Enumeration“ is disabled. Can I do anything to help you find out, what is going on here? I view this web site on a daily basis. And I swear, it worked until yesterday.


   Regards,

   Frank

Von: Andrew [mailto:notifications@github.com]
Gesendet: Mittwoch, 6. Dezember 2017 21:03
An: andryou/scriptsafe <scriptsafe@noreply.github.com>
Cc: xformerfhs <frank.schwab@live.de>; Author <author@noreply.github.com>
Betreff: Re: [andryou/scriptsafe] Page "http://franom.fraport.de/franom.php" no longer shows planes after update to 1.0.9.2 (#293)

Thanks for the report, do you have "Block Browser Plugin Enumeration" under
Fingerprint Protection enabled?

**Comment by xformerfhs** [comment]
Hello, Andrew,

   here are the objects that are allowed:

[cid:image001.png@01D36ED6.D2E7CE20]

   Regards,

   Frank

Von: Andrew [mailto:notifications@github.com]
Gesendet: Mittwoch, 6. Dezember 2017 21:03
An: andryou/scriptsafe <scriptsafe@noreply.github.com>
Cc: xformerfhs <frank.schwab@live.de>; Author <author@noreply.github.com>
Betreff: Re: [andryou/scriptsafe] Page "http://franom.fraport.de/franom.php" no longer shows planes after update to 1.0.9.2 (#293)

Thanks for the report, do you have "Block Browser Plugin Enumeration" under
Fingerprint Protection enabled?

**Comment by AdamF001** [comment]
Hi, Just a follow up, I have checked out the page aswell and with Scriptsafe enabled the planes are visible but only the previous day shows and the days before, current day shows no planes, 

With Scriptsafe disabled I still cannot see planes for the current day but the previous days do show planes, I have Clipboard Interference turned on and allowed on the website.

Allowed Items
--
maps.google.com (12)
google.com (2)
maps.googleapis.com (3)
franom.fraport.de (Clipboard Interference)

Google Chrome is up to date
Version 63.0.3239.84 (Official Build) beta (64-bit)
Windows 10 Version 1709 Build 16299.98

**Comment by xformerfhs** [comment]
Hi, Andrew,

   the same for me. I just checked with MS Edge and Firefox and the planes for the current day do not show up there, as well. Seems to be a bug of the web site.

   So this is obviously just a coincidence. I should have checked that before, but I have been using this web site for years and never have seen anything like this. So this was my bad. Sorry for bothering you and thanks for your very fast response.


   Best regards,

   Frank

Von: Adam F. [mailto:notifications@github.com]
Gesendet: Mittwoch, 6. Dezember 2017 21:21
An: andryou/scriptsafe <scriptsafe@noreply.github.com>
Cc: xformerfhs <frank.schwab@live.de>; Author <author@noreply.github.com>
Betreff: Re: [andryou/scriptsafe] Page "http://franom.fraport.de/franom.php" no longer shows planes after update to 1.0.9.2 (#293)


Hi, Just a follow up, I have checked out the page aswell and with Scriptsafe enabled the planes are visible but only the previous day shows and the days before, current day shows no planes,

With Scriptsafe disabled I still cannot see planes for the current day but the previous days do show planes, I have Clipboard Interference turned on and allowed on the website.

Allowed Items

maps.google.com (12)
google.com (2)
maps.googleapis.com (3)
franom.fraport.de (Clipboard Interference)

Google Chrome is up to date
Version 63.0.3239.84 (Official Build) beta (64-bit)
Windows 10 Version 1709 Build 16299.98
